datalake/src/main/java/com/sequenceiq/datalake/flow/salt/update/handler/SaltUpdateWaitHandler.java

```diff
@@ -0,0 +1,73 @@
+package com.sequenceiq.datalake.flow.salt.update.handler;
+
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.datalake.entity.SdxCluster;
+import com.sequenceiq.datalake.flow.salt.update.event.SaltUpdateFailureResponse;
+import com.sequenceiq.datalake.flow.salt.update.event.SaltUpdateWaitRequest;
+import com.sequenceiq.datalake.flow.salt.update.event.SaltUpdateWaitSuccessResponse;
+import com.sequenceiq.datalake.service.sdx.PollingConfig;
+import com.sequenceiq.datalake.service.sdx.SdxService;
+import com.sequenceiq.datalake.service.sdx.flowwait.SdxWaitService;
+import com.sequenceiq.datalake.service.sdx.flowwait.exception.SdxWaitException;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+
+import reactor.bus.Event;
+
+@Component
+public class SaltUpdateWaitHandler extends ExceptionCatcherEventHandler<SaltUpdateWaitRequest> {
```

Comment:
please cover this with UT

Reply:
Thanks, I forget to add those. Added it now.